recipes-core/images/intel-aero-image.bb

```diff
@@ -35,6 +35,7 @@ IMAGE_INSTALL += "rng-tools"
 
 # Build tools
 IMAGE_INSTALL += "packagegroup-core-buildessential"
+IMAGE_INSTALL += "catkin"
```

Comment:
should be below, together with the ROS tools. Please also provide the image size before and after.  After this change can you run catkin_make in the image?